deploy/chart/values.yaml

```diff
@@ -19,9 +19,12 @@ olm:
   nodeSelector:
     beta.kubernetes.io/os: linux
   resources:
+    limits:
+      cpu: 20m
```

Comment:
1m = 1/1000 CPU core. Are we sure that our operators top out under 1/50th of a CPU core?